Source/Private/GenerateParameterFile.ps1

```diff
@@ -68,10 +71,20 @@ function GenerateParameterFile {
     
     switch ($PSCmdlet.ParameterSetName) {
         'FromFile' {
-            Out-File -InputObject $ConvertedToJson -FilePath "$($file.DirectoryName)\$filename.parameters.json" -WhatIf:$WhatIfPreference
+            if($AsHashtable.IsPresent) {
+                $ConvertedToJson | ConvertToHashtable -Ordered
```

Comment:
This does not work. `ConvertToHashtable` does not expect JSON as input. This will only output the parameter file to the terminal.